public/js/ticTacToe.js

```diff
@@ -3,6 +3,61 @@ const playerTurn = 'X'
 const Xmoves = []
 const Omoves = []
 
-const boardGame = document.querySelector('.board')
+const boardGame = document.querySelector('#board')
 const resetButton = document.querySelector('#reset')
+const divBoxes = document.querySelectorAll('div.box')
+const winningCombos = [[1,2,3], [4,5,6], [7,8,9], [1,4,7], [1,5,9], [3,5,7], [3,6,9], [2,5,8]]
 
+const switchTurn = function() {
+  if(playerTurn === 'X') {
+    playerTurn = 'O'
+  } else {
+    playerTurn = 'X'
+  }
+}
+
+for(let i = 0; i < divBoxes.length; i++) {
+  divBoxes[0].getAttribute("id")
```

Comment:
This should probably be:
```
let divBoxIds = []
for( let i = 0; i < divBoxes.length; i++) {
  let id = divBoxes[i].getAttribute("id")
  divBoxIds.push( id )
}
```
two things happening. You're grabbing the attributes of those id's, but then not doing anything with them. In this instance, I assumed you wanted to store them in an array for later use, so I create an array on the outside of the loop (because of scoping. Google or ask me and I can elaborate) and then push each one of those id's into the array.

The second thing to notice: the '0' is 'i', because we want to select each item inside divBoxes, not just the first one, or if you did want to select only the first one, you wouldn't need to loop over them.

Another way, also not necessarily better or worse in this particular context:
```
let divBoxIds = []
divBoxes.forEach( function(box) {
  let id = box.getAttribute("id")
  divBoxIds.push( id )
}
```

and finally 3rd alternative of same thing:
```
let divBoxIds = []
for(let box of divBoxes) {
  let id = box.getAttribute("id")
  divBoxIds.push( id )
}
```

Or probably the best way, but most complicated looking:
```
let divBoxIds = divBoxes.reduce( function(accumulator, element) {
  accumulator.push( element.getAttribute("id") )
  return accumulator
}, [] )
```